szperacz/run.py

```diff
@@ -64,8 +64,12 @@ def log_to_file(files):
 def headless(search_path):
     logger.info('Szperacz in headless form')
     files = get_files(search_path)
+    logger.error('start error')                      # test
     log_to_console(files)
     log_to_file(files)
+    logger.error('log to file error')                # test
+    logger.error('parchacz')                         # test
+    logger.get_log_level()
```

Comment:
These `test` entries and `get_log_level` call do not belong here.

Let's create a proper test in `tests/` directory for the logging class.